# Manuel Belmadani
*Data Scientist with experience in bioinformatics consulting, academic research and software development.*

- Background 
  - [Curriculum Vitae](https://docs.google.com/document/d/1PqUhHGP_DJcX1H7v0MoNYXHpu28rx2h0HPLMchafZxs/edit?usp=sharing)
  - MSc. Computer Science specialized in Bioinformatics, University of Ottawa (2012-2015)
  - BSc. Computer Science specializion with Honours, University of Ottawa (2007-2012)
- Contact: 
  - [LinkedIn](https://www.linkedin.com/in/manuel-belmadani-62780124/)
  - E-mail: `manuel.belmadani` ![](https://i.imgur.com/OhJr77y.png) `pm`![](https://i.imgur.com/LDf3iFQ.png)`me`


### Overview

Advising and design of bioinformatics solutions and analysis plans for research projects. Comfortable in applying computional data analysis to understand data from genomics, proteomics, transcriptomics and model organism assays or experiments. The core of my recent work has mostly been combinations of statistics and visualization, software development, Linux-based computing solutions, databases design and web technologies. I enjoy working on research projects and can tackle software engineering challenges along the way.

#### Services Offered
Generally, I'm open to discuss a research problem if:
1) I can make the time given my existing engagements and contracts
2) The research question is sound and a good fit for my skill set

Feel free to use one of the contact methods above to connect with me. Any of the topics listed below are areas I'm comfortable advising in:

Key areas of expertise:   
`bioinformatics`,  
`statistical analysis`,  
`cloud application development`,  
`genomics`,  
`proteomics`,  
`transcriptomics`,  
`variant calling`,  
`experimental data analysis`,  
`genotype-disease analysis`


### Publications
[Multi-model functionalization of disease-associated PTEN missense mutations identifies multiple molecular mechanisms underlying protein dysfunction. Nat. Commun. 11, 2073 (2020)](https://doi.org/10.1038/s41467-020-15943-0) Post, K. L., **Belmadani, M.**, Ganguly, P., Meili, F., Dingwall, R., McDiarmid, T. A., Meyers, W. M., Herrington, C., Young, B. P., Callaghan, D. B., Rogic, S., Edwards, M., Niciforovic, A., Cau, A., Rankin, C. H., O’Connor, T. P., Bamji, S. X., Loewen, C. J. R., Allan, D. W., Pavlidis, P. & Haas, K. 

[Systematic phenomics analysis of autism-associated genes reveals parallel networks underlying reversible impairments in habituation. Proc. Natl. Acad. Sci. U. S. A. 117, 656–667 (2020)](https://doi.org/10.1073/pnas.1912049116) McDiarmid, T. A., **Belmadani, M.**, Liang, J., Meili, F., Mathews, E. A., Mullen, G. P., Hendi, A., Wong, W.-R., Rand, J. B., Mizumoto, K., Haas, K., Pavlidis, P. & Rankin, C. H. 

[VariCarta: A Comprehensive Database of Harmonized Genomic Variants Found in Autism Spectrum Disorder Sequencing Studies. Autism Res. 12, 1728–1736 (2019)](https://doi.org/10.1002/aur.2236) **Belmadani, M.**, Jacobson, M., Holmes, N., Phan, M., Nguyen, T., Pavlidis, P. & Rogic, S. 

[Whole genome sequencing and variant discovery in the ASPIRE autism spectrum disorder cohort. Clin. Genet. (2019) doi:10.1111/cge.13556](https://dx.doi.org/10.1111/cge.13556) Callaghan, D. B., Rogic, S., Tan, P. P. C., Calli, K., Qiao, Y., Baldwin, R., Jacobson, M., **Belmadani, M.**, Holmes, N., Yu, C., Li, Y., Li, Y., Kurtzke, F.-E., Kuzeljevic, B., Yu, A., Hudson, M., McNaughton, A. J. M., Xu, Y., Dionne-Laporte, A., Girard, S., Liang, P., Separovic, E. R., Liu, X., Rouleau, G., Pavlidis, P. & Lewis, M. E. S. 

[Assessing transcriptome quality in patch-seq datasets. Front. Mol. Neurosci. 11, 363 (2018)](https://dx.doi.org/10.3389%2Ffnmol.2018.00363) Tripathy S., Toker L., Bomkamp C., Mancarci O., **Belmadani M.** & Pavlidis P. 

[MotifGP: Using multi-objective evolutionary computing for mining network expressions in DNA sequences. in 2016 IEEE Conference on Computational Intelligence in Bioinformatics and Computational Biology (CIBCB) 1–6 (2016) doi:10.1109/CIBCB.2016.7758133](https://dx.doi.org/10.1109/CIBCB.2016.7758133) **Belmadani M.** & Turcotte M. 

[A comprehensive analysis of 3’ end sequencing data sets reveals novel polyadenylation signals and the repressive role of heterogeneous ribonucleoprotein C on cleavage and polyadenylation. Genome Res. (2016) doi:10.1101/gr.202432.115](https://dx.doi.org/10.1101/gr.202432.115) Gruber A. J., Schmidt R., Gruber A. R., Martin G., Ghosh S., **Belmadani M.**, Keller W. & Zavolan M. 

### Links

- [List of publications](https://scholar.google.com/citations?user=CaXlIcUAAAAJ&hl=en&oi=ao)
- [Github](https://github.com/mbelmadani/)
- [ORCID](https://orcid.org/0000-0002-5820-5979)

Some web applications I've worked on:
- [VariCarta](https://varicarta.msl.ubc.ca/) - A database of harmonized ASD genomic variants
- [RDMM - Rare Diseases: Models & Mechanisms Network](https://register.rare-diseases-catalyst-network.ca/) - A register for rare disease research and model organism collaboration.
- [PolyASite](https://polyasite.unibas.ch/) - A web atlas of 3′ end processing sites.
